bondcpp/src/bond.cpp

```diff
@@ -70,75 +70,87 @@ static std::string makeUUID()
 
 Bond::Bond(
   const std::string & topic, const std::string & id,
-  rclcpp_lifecycle::LifecycleNode::SharedPtr nh,
-  std::function<void(void)> on_broken,
-  std::function<void(void)> on_formed)
-: bondsm_(new BondSM(this)),
+  rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_base,
+  rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr node_logging,
+  rclcpp::node_interfaces::NodeParametersInterface::SharedPtr node_params,
+  rclcpp::node_interfaces::NodeTimersInterface::SharedPtr node_timers,
+  rclcpp::node_interfaces::NodeTopicsInterface::SharedPtr node_topics,
```

Comment:
Kind of overkill, but can you add `#include <rclcpp/rclcpp.hpp>` at the top of this file?